From b9082de314d9e12d6ff436a2606f4428c48c290f Mon Sep 17 00:00:00 2001
From: Olivier XILLO <oxdev@mailo.com>
Date: Mon, 24 Feb 2025 19:57:28 +0100
Subject: [PATCH] fix: CMAKE install

---
 src/ed247/CMakeLists.txt | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/src/ed247/CMakeLists.txt b/src/ed247/CMakeLists.txt
index 16d1293..dd5146c 100644
--- a/src/ed247/CMakeLists.txt
+++ b/src/ed247/CMakeLists.txt
@@ -95,14 +95,6 @@ endmacro(_generate_xsd_header _XSD_PATH _XSD_FILENAME _HEADER_FILEPATH _VAR_NAME
 _generate_xsd_header(${CMAKE_CURRENT_LIST_DIR}/xsd ED247A_ECIC.xsd ${CMAKE_CURRENT_BINARY_DIR}/ed247_xsd.h xsd_schema)
 
 # Install required stuff
-set(INSTALL_LIBDIR lib)
-if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
-  set(INSTALL_LIBDIR bin)
-endif()
-
-install(TARGETS ed247 ed247_static
-                                     DESTINATION ${PLATFORM_INSTALL_SUBDIR}${INSTALL_LIBDIR}
-  PUBLIC_HEADER                      DESTINATION ${PLATFORM_INSTALL_SUBDIR}include
-)
+install(TARGETS ed247 ed247_static)
 install(FILES ./xsd/ED247A_ECIC.xsd  DESTINATION doc/xsd)
 install(FILES ./xsd/ED247A_ECD.xsd   DESTINATION doc/xsd)
-- 
2.48.1

